### PR TITLE
Feature/data 13846 load apple music to db

### PR DESCRIPTION
**Pull request by ben-xo**
* Add schema support to EPFImporter
* Add some TODOs for future work
* Increase batch size for better performance in GKE.

### CODE REVIEW

File: EPFIngester.py [hunk 1]
```diff
@@ -74,9 +74,19 @@ def __init__(self,
...
-        self.tableName = (pref + self.fileName).replace("-", "_") #hyphens aren't allowed in table names
+        self.tableName = self.fileName.replace("-", "_") #hyphens aren't allowed in table names
         self.tableName = self.tableName.split(".")[0]
+
+        pref = ""
+        if tablePrefix:
+            # add _ separator to prefix unless the prefix is a schema or empty
+            pref = ("%s_" % tablePrefix if len(tablePrefix) and tablePrefix[-1] != "." else tablePrefix)
+
+        if pref.split(".")[0] != pref:
```
Comment (Hitobat) [inline EPFIngester.py:85]: I think `if "." not in pref` is more idiomatic here.

File: EPFIngester.py [hunk 2]
```diff
@@ -74,9 +74,19 @@ def __init__(self,
...
+            # looks like there's a schema.
+            self.tableSchema, pref = pref.split(".")
```
Comment (Hitobat) [inline EPFIngester.py:87]: `self.tableSchema` doesn't look to be defined at all if this branch is missed.

**Review by Hitobat — CHANGES_REQUESTED** [review]
I believe there's a small bug for the tableSchema var.
If the prefix doesn't include . then the reference to self.tableSchema when setting the search_path will result in an AttributeError exception.

File: EPFIngester.py (hunk 2 above)
Comment (ben-xo) [reply EPFIngester.py:87]: that's fine; the search_path checks if it's set.

Comment (ben-xo) [reply EPFIngester.py:88]: FIxed in 4666799

**Comment by ben-xo** [comment]
> I believe there's a small bug for the tableSchema var.
> If the prefix doesn't include . then the reference to self.tableSchema when setting the search_path will result in an AttributeError exception.

Fixed in 4666799

File: EPFIngester.py (hunk 1 above)
Comment (ben-xo) [reply EPFIngester.py]: Fixed in 4666799

**Review by Hitobat — APPROVED** [review]

**Pull request merged**